tests/fuzz/fuzz-bolt11.c

```diff
@@ -126,6 +126,8 @@ void run(const uint8_t *data, size_t size)
 	const u5 *sigdata;
 	bool have_n = false;
 	char *fail;
+
+	dev_bech32_nocsum = true;
```

Comment:
This is the wrong fuzz test to enable the flag.  We should do this during `init` in `fuzz-bolt11-decode.c` instead.